rma/wizard/rma_re_delivering_views.xml

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Copyright 2020 Tecnativa - Ernesto Tejeda
+     License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl). -->
+<odoo>
+    <record id="rma_redelivering_wizard_view_form" model="ir.ui.view">
+        <field name="name">rma.re.delivering.wizard.form</field>
+        <field name="model">rma.re.delivering.wizard</field>
+        <field name="arch" type="xml">
+            <form>
+                <group>
+                    <group>
+                        <field name="operation" />
+                    </group>
+                    <group>
+                        <field name="replace_product_id"
+                               attrs="{'invisible': [('operation', '!=', 'replace')], 'required': [('operation', '=', 'replace')]}"/>
+                    </group>
+                </group>
+                <footer>
+                    <button name="action_re_deliver" string="Re-deliver" type="object" class="btn-primary"/>
+                    <button string="Cancel" class="btn-secondary" special="cancel" />
+                </footer>
+            </form>
+        </field>
+    </record>
+    <act_window id="rma_re_delivering_wizard_action"
+                name="Re-deliver"
```

Comment:
Is re-deliver the right term?

Reply:
I looked in some dictionaries and it seemed a correct term for this (for example: https://en.wiktionary.org/wiki/redeliver), but I have already changed it to avoid confusion.